tests/cleos_set_code_test.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import random
+from TestHarness import Account, Cluster, TestHelper, Utils, WalletMgr
+
+###############################################################
+# cleos_set_code_test
+#
+# Test Cleos "set code" and "set abi" commands
+#
+###############################################################
```

Comment:
This test is fine, but I think you could have just added it to `nodeos_run_test.py` to save on test run times.

Reply:
Thanks! I will change to use `nodeos_run_test.py` to save test setup time.